src/js/load-package.ts

```diff
@@ -441,21 +440,46 @@ export async function loadPackage(
           loadedPackages[name] = channel;
         })
         .catch((err) => {
-          console.warn(err);
-          failed[name] = err;
+          failed.set(name, err);
         });
     }
     await Promise.all(Object.values(packageInstallPromises));
 
+    // Retry failed packages after all dependencies are downloaded.
+    const installPromisesRetry: { [name: string]: Promise<void> } = {};
+    const toLoadRetry = new Map(
+      [...toLoadShared, ...toLoad].filter(([name, channel]) => failed.has(name))
+    );
```

Comment:
The other way here is a bit more efficient:
```suggestion
    const toLoadRetry = new Map(
      failed.keys().map(k => [k, toLoadShared[k] || toLoad[k]])
    );
```